src/main/java/io/stargate/sgv2/jsonapi/service/shredding/model/DocValueHasher.java

```diff
@@ -195,12 +198,14 @@ public DocValueHash getHash(Object value) {
       return numberValue((BigDecimal) value).hash();
     } else if (value instanceof Boolean) {
       return booleanValue((Boolean) value).hash();
-    } else if (value instanceof Date) {
-      return timestampValue((Date) value).hash();
+    } else if (value instanceof Instant) {
+      return timestampValue(new Date(((Instant) value).toEpochMilli())).hash();
```

Comment:
Does this change the date handling behavior? Doesn't seem obviously related to bridge removal.

Reply:
timestamp cql data type uses Instant java type.